tools/clang/test/HLSLFileCheck/validation/WaveSizeAttrNotAllowed.hlsl

```diff
@@ -0,0 +1,9 @@
+// RUN: %dxc -T lib_6_7 %s | FileCheck %s
+
+// CHECK: error: attribute numthreads only valid for CS/MS/AS.
```

Comment:
That would be ideal, but #5367 prevents this. There will likely be a large migration of tests over to verifier tests, so I wonder if there's a way to mark these tests so that they're not forgotten.

Reply:
If you put this test into `tools/clang/test/SemaHLSL/...` instead of under `HLSLFileCheck` it can run as a lit test using the `dxc -verify` flag which respects the `-T` flag.